torch/onnx/symbolic_helper.py

```diff
@@ -800,6 +800,35 @@ def _interpolate_get_scales_and_mode(g, input, size, scale_factor, mode, align_c
     return scale_factor, mode
 
 
+def _argmin_argmax_helper(g, input, dim, keepdim, op_name):
+    keepdim = _parse_arg(keepdim, "i")
+
+    def op_wrapper(input, axis_i, keepdims_i):
```

Comment:
I would avoid nested functions and pull this out

Reply:
Can you elaborate? I think placing it inside this helper would be more concise than pulling it out.

Reply:
Nested functions are harder to test and maintain, especially when they are a closure. We should eventually get rid of all the opset version conditioning. This particular case is ok though I think. (resolved)